src/Features/Core/Portable/Wrapping/InitializerExpression/InitializerExpressionCodeActionComputer.cs

```diff
@@ -0,0 +1,330 @@
+                return indentString;
+            }
+
+            private string GetSingleIdentation()
+            {
+                // Insert a newline after the open token of the list.  Then ask the
+                // ISynchronousIndentationService where it thinks that the next line should be
+                // indented.
+                var openToken = _listSyntax.GetFirstToken();
+
+                return GetSmartIndentationAfter(openToken);
+            }
+
+            protected override async Task<ImmutableArray<WrappingGroup>> ComputeWrappingGroupsAsync()
+            {
+                var result = ArrayBuilder<WrappingGroup>.GetInstance();
+                await AddWrappingGroups(result).ConfigureAwait(false);
+                return result.ToImmutableAndFree();
+            }
+
+            private async Task AddWrappingGroups(ArrayBuilder<WrappingGroup> result)
+            {
+                result.Add(await GetWrapEveryGroupAsync().ConfigureAwait(false));
+                result.Add(await GetUnwrapGroupAsync().ConfigureAwait(false));
+                result.Add(await GetWrapLongGroupAsync().ConfigureAwait(false));
+            }
+
+            private async Task<WrappingGroup> GetWrapEveryGroupAsync()
+            {
+                var parentTitle = Wrapper.Wrap_every_item;
+
+                var codeActions = ArrayBuilder<WrapItemsAction>.GetInstance();
+
+                codeActions.Add(await GetWrapEveryNestedCodeActionAsync(
+                    parentTitle, WrappingStyle.WrapFirst_IndentRest).ConfigureAwait(false));
+
+                // See comment in GetWrapLongTopLevelCodeActionAsync for explanation of why we're
+                // not inlinable.
+                return new WrappingGroup(isInlinable: false, codeActions.ToImmutableAndFree());
+            }
+
+            private async Task<WrapItemsAction> GetWrapEveryNestedCodeActionAsync(
+                string parentTitle, WrappingStyle wrappingStyle)
+            {
+                var indentationTrivia = GetIndentationTrivia(wrappingStyle);
+
+                var edits = GetWrapEachEdits(wrappingStyle, indentationTrivia);
+                var title = GetNestedCodeActionTitle(wrappingStyle);
+
+                return await TryCreateCodeActionAsync(edits, parentTitle, title).ConfigureAwait(false);
+            }
+
+            private SyntaxTrivia GetIndentationTrivia(WrappingStyle wrappingStyle)
+            {
+                return wrappingStyle == WrappingStyle.UnwrapFirst_AlignRest
+                    ? _afterOpenTokenIndentationTrivia
+                    : _singleIndentationTrivia;
+            }
+
+            private string GetNestedCodeActionTitle(WrappingStyle wrappingStyle)
+                => wrappingStyle switch
+                {
+                    WrappingStyle.WrapFirst_IndentRest => Wrapper.Indent_all_items,
+                    _ => throw ExceptionUtilities.UnexpectedValue(wrappingStyle),
+                };
```

Comment:
what's going on here?

Reply:
This was something I copied over from the other file I was using as a reference. It was to switch between different wrapping styles, but since initializers only support one now I can go ahead and remove this function.

Reply:
This was a method I copied over from the other class. It covered more wrapping styles, but initializers only have the one wrapping style. I can go ahead and remove it since it isn't doing much anymore.